Assign Pick List

Irisvella opened the pull request:
#89 #88 #90 
- API to assign a pick list to a staff 
- Dropdown menu to select and assign staff

Irisvella committed:
#89 #90 Assign Staff via Dropdown menu

- Can assign Staff via search
- Links order to them
- staff view dashboard

To do:
-tests
- assign only staff
#89 #90

Irisvella committed:
filters the list to only view staff role #88 #90

flo351 reviewed (approved):
Nice work on the API and dropdown for assigning staff! Filtering by role looks great.

Irisvella committed:
Staff can view their assigned pick list

- Staff can view their assigned pick list. API + Frontend #91 #92 #88
- Push "check picked item" to next iteration as it depends on barcodes and item labels.

Irisvella commented:
Also added staff view of assigned inventory pick list.
As a staff, you can view all pick lists assigned to you. You can also view all the items inside the pick list. 
Checklist for items is pushed to next iteration probably 

![image](https://github.com/user-attachments/assets/b13f326b-c40e-4022-b2aa-428c2e481013)

NourHadieHassoun reviewed (approved):
Good work on the added staff view.!

Irisvella committed:
Merge branch 'main' into assign-picklist-andy

Irisvella committed:
Fixed merge conflicts, fixed "back" button according to role
